.github/workflows/schedule-jira-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync Security Hub findings and Jira issues
+
+on:
+  workflow_dispatch: # for testing and manual runs
+  schedule:
+    - cron: '25 1 * * *' # daily at 6:25 am EST
+
+permissions:
+  id-token: write
+
+jobs:
+  sync:
+    name: Run sync
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
```

Comment:
Question: It doesn't seem like we're using any files from the repo in latter steps. Can we remove this one?

Reply:
I suggest to keep it just incase.